Mathlib/Topology/Algebra/SeparationQuotient.lean

```diff
@@ -195,6 +195,40 @@ instance instGroup [Group G] [TopologicalGroup G] : Group (SeparationQuotient G)
 instance instCommGroup [CommGroup G] [TopologicalGroup G] : CommGroup (SeparationQuotient G) :=
   surjective_mk.commGroup mk mk_one mk_mul mk_inv mk_div mk_pow mk_zpow
 
+/-- Neighborhoods in the quotient are precisely the map of neighborhoods in the prequotient. -/
+theorem nhds_eq [AddCommGroup G] [TopologicalAddGroup G] (x : G) :
+    nhds (mk x) = Filter.map mk (nhds x) := by
+  apply le_antisymm ((SeparationQuotient.isOpenMap_mk).nhds_le x) continuous_quot_mk.continuousAt
+
+/-- `SeparationQuotient.mk` as a `GroupHom`. -/
+@[to_additive (attr := simps) "`SeparationQuotient.mk` as an `AddGroupHom`."]
```

Comment:
```suggestion
/-- `SeparationQuotient.mk` as a `MonoidHom`. -/
@[to_additive (attr := simps) "`SeparationQuotient.mk` as an `AddMonoidHom`."]
```
since there is no such thing as what you wrote in mathlib